### PR TITLE
0216 allow starting two nodes from the same boot script

### PR DESCRIPTION
**Pull request by zmstone**
This is a follow up fix for an issue found after: https://github.com/emqx/emqx/pull/9961 is merged.

### CODE REVIEW

**Review by zhongwencool — DISMISSED** [review]

File: scripts/test/start-two-nodes-in-host.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    EMQX_CLUSTER__DISCOVERY_STRATEGY=static \
+    EMQX_LOG__FILE_HANDLERS__DEFAULT__LEVEL="${EMQX_LOG__FILE_HANDLERS__DEFAULT__LEVEL:-debug}" \
+    EMQX_LOG__FILE_HANDLERS__DEFAULT__FILE="$LOG1/emqx.log" \
+    EMQX_NODE_NAME='emqx1@127.0.0.1' \
+    EMQX_LOG_DIR="$LOG1" \
+    EMQX_NODE__DATA_DIR="$DATA1" \
+    EMQX_LISTENERS__TCP__DEFAULT__BIND='0.0.0.0:1881' \
+    EMQX_LISTENERS__SSL__DEFAULT__BIND='0.0.0.0:8881' \
+    EMQX_LISTENERS__WS__DEFAULT__BIND='0.0.0.0:8081' \
+    EMQX_LISTENERS__WSS__DEFAULT__BIND='0.0.0.0:8084' \
```
Comment (ieQu1) [inline scripts/test/start-two-nodes-in-host.sh:46]:
```suggestion
    EMQX_LISTENERS__TCP__DEFAULT__BIND='0.0.0.0:1881' \
    EMQX_LISTENERS__SSL__DEFAULT__BIND='127.0.0.1:8881' \
    EMQX_LISTENERS__WS__DEFAULT__BIND='127.0.0.1:8081' \
    EMQX_LISTENERS__WSS__DEFAULT__BIND='127.0.0.1:8084' \
```

?

File: scripts/test/start-two-nodes-in-host.sh [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    EMQX_LOG__FILE_HANDLERS__DEFAULT__LEVEL="${EMQX_LOG__FILE_HANDLERS__DEFAULT__LEVEL:-debug}" \
+    EMQX_LOG__FILE_HANDLERS__DEFAULT__FILE="$LOG2/emqx.log" \
+    EMQX_NODE_NAME='emqx2@127.0.0.1' \
+    EMQX_LOG_DIR="$LOG2" \
+    EMQX_NODE__DATA_DIR="$DATA2" \
+    EMQX_LISTENERS__TCP__DEFAULT__BIND='0.0.0.0:1882' \
+    EMQX_LISTENERS__SSL__DEFAULT__BIND='0.0.0.0:8882' \
+    EMQX_LISTENERS__WS__DEFAULT__BIND='0.0.0.0:8082' \
+    EMQX_LISTENERS__WSS__DEFAULT__BIND='0.0.0.0:8085' \
+    EMQX_DASHBOARD__LISTENERS__HTTP__BIND='0.0.0.0:18082' \
```
Comment (ieQu1) [inline scripts/test/start-two-nodes-in-host.sh:62]:
```suggestion
    EMQX_LISTENERS__TCP__DEFAULT__BIND='127.0.0.2:1881' \
    EMQX_LISTENERS__SSL__DEFAULT__BIND='127.0.0.2:8882' \
    EMQX_LISTENERS__WS__DEFAULT__BIND='127.0.0.2:8082' \
    EMQX_LISTENERS__WSS__DEFAULT__BIND='127.0.0.2:8085' \
    EMQX_DASHBOARD__LISTENERS__HTTP__BIND='127.0.0.2:18082' \
```

**Comment by ieQu1** [comment]
Listen on a single IP ?

**Comment by zmstone** [comment]
> Listen on a single IP ?

Good idea. Fixed.

**Review by ieQu1 — APPROVED** [review]

**Pull request merged**